package.json

```
@@ -201,6 +201,11 @@
 				  "name"
 				]
 			  }
+			},
+			"bitbake.disableConfigModification":{
+				"type": "boolean",
+				"default": false,
+				"description": "Turn on this setting to stop this extension from modifying the vscode configuration of the workspace. Note that this will also disable some features related to embedded python and bash code."
```

```suggestion
				"description": "Turn on this setting to stop this extension from modifying the VS Code configuration of the workspace. Note that this will also disable some features related to embedded Python and Shell code."
```

Just to be consistent with the description of the other options.

---

Thanks, done